src/css/flexbox.css

```diff
@@ -0,0 +1,78 @@
+.header {
+    display: flex;
+    background-color: #f0f0f0;
+}
+
+.header .name {
+    flex: 1;
+    padding: 16px;
+    position: relative;
+    font-size: 24px;
+}
+
+.header .img1 {
+    position: relative;
+    flex: 2;
+    padding: 16px;
+}
+
+.menu ul {
+    background-color: #C0C0C0;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+    display: flex;
+}
+
+a {
+    text-decoration: none;
+    color: black
+}
+
+.menu .item {
+    position: relative;
+    color: black;
+    padding: 16px;
+}
+
+footer {
+    background: #C0C0C0;
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    display: flex;
+    padding-top: 16px;
+    padding-left: 16px;
+    padding-bottom: 16px;
+}
+
+.h2 {
+    padding-top: 24px;
+    padding-left: 88px;
+    font-weight: bold;
+    font-size: 18px
+}
+
+.text {
+    display: flex;
+    flex-wrap: wrap;
+	align-items: center;
+    padding-right: 88px;
+    padding-left: 78px;
+}
+
+.text1 {
+    margin: 2%;
+}
+
+.left {
+    display: flex;
+    position: relative;
+    flex: 7;
```

Comment:
thats a super high number, do you know what that means?

Reply:
It means how much the box grows, shrinks or the length (basis) in its container. I was being a little hacky to make the paragraph in the next container move over.